src/components/editions/byline/index.tsx

```diff
@@ -1,15 +1,15 @@
 // ----- Imports ----- //
-import type { SerializedStyles } from '@emotion/core';
-import { css } from '@emotion/core';
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
 import { neutral, remSpace } from '@guardian/src-foundations';
-import { from } from '@guardian/src-foundations/mq';
-import { border } from '@guardian/src-foundations/palette';
-import { body, headline } from '@guardian/src-foundations/typography';
 import type {
 	FontStyle,
 	FontWeight,
 	LineHeight,
-} from '@guardian/src-foundations/typography/types';
+} from '@guardian/src-foundations/dist/types/typography/types';
```

Comment:
Does `src-foundations` no longer export these?

Reply:
@SiAdcock is aware of these, I promised him I'll make a list of types we depend on that aren't exported and let him know so they can update their packages accordingly.